RHIDP-6410 - added steps  to Azure authentication procedures (#1168)
redhat-developer/red-hat-developers-documentation-rhdh: modules/authentication/proc-provisioning-users-from-microsoft-azure-to-the-software-catalog.adoc

* You have xref:enabling-authentication-with-microsoft-azure[enabled authentication with Microsoft Azure].

.Procedure
. link:{installing-and-viewing-plugins-book-url}[Enable the `backstage-plugin-catalog-backend-module-msgraph-dynamic` plugin].
+
.`dynamic-plugins.yaml` file fragment

> ----

Author: ```suggestion
[code,yaml]
----
```

* You have xref:enabling-authentication-with-microsoft-azure[enabled authentication with Microsoft Azure].

.Procedure

> 

Author: ```suggestion
```

+
.`dynamic-plugins.yaml` file fragment
[code,yaml]
----
plugins:
  - package: './dynamic-plugins/dist/backstage-plugin-catalog-backend-module-msgraph-dynamic'
    disabled: false
----

> . To enable Microsoft Azure member discovery, edit your custom {product-short} ConfigMap, such as `app-config-rhdh`, and add following lines to the `{my-app-config-file}` content:

Reviewer: ```suggestion
. To enable Microsoft Azure member discovery, edit your custom {product-short} ConfigMap, such as `app-config-rhdh`, and add the following lines to the `{my-app-config-file}` content:
```

> . To enable Microsoft Azure member discovery, edit your custom {product-short} ConfigMap, such as `app-config-rhdh`, and add the following lines to the `{my-app-config-file}` content:

Author: ```suggestion
. To enable {azure-brand-name} member discovery, edit `{my-app-config-file}`, your custom {product-short} configuration file::
```

        tenantId: ${AUTH_AZURE_TENANT_ID}
        clientId: ${AUTH_AZURE_CLIENT_ID}
        clientSecret: ${AUTH_AZURE_CLIENT_SECRET}

>         schedule:

Reviewer 2: It'd be nice to add a comment about being careful of setting this frequency, especially for large organizations.  Something like the upstream docs [here](https://backstage.io/docs/integrations/azure/org#installation)

`tenandId`, `clientId` and `clientSecret`::
Use the {product-short} application information you created in Microsoft Azure and configured in OpenShift as secrets.

`schedule`::

`frequency`:::
Enter the schedule frequency as cron, ISO duration, or human duration as used in code.

`timeout`:::
Enter the schedule timeout as ISO duration or human duration as used in code.

`initialDelay`:::
Enter the schedule initial delay as ISO duration or human duration as used in code.
+

> TIP: In a large organization, this plugin can take a long time, therefore avoid setting low frequency or timeout when importing a large amount of users and groups for the first try.

Reviewer: ```suggestion
TIP: In a large organization, this plugin can take a long time. Therefore, avoid setting a low frequency or timeout when importing a large number of users and groups for the first time.
```

> In a large organization, this plugin can take a long time to? 

To provide more clarity to users, consider adding some context here.
